src/style.css

```diff
@@ -0,0 +1,100 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf)  format("turetype");
+  font-style: normal;
+  font-weight: 800;
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 300;
+  font-style: normal;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+.search-form {
+  margin-top: 20px;
+}
+
+.input {
+  font-family: "Avenir", serif;
```

Comment:
it would be better to add font-family to HTML as separate style
```
html {
  font-family: Avenir, sans-serif;
}
```
and here you can just write `font-family: inherit;`. please check other places as well